starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,53 @@
+.topclass {
+    background-image: url("../images/apple-pie.jpg");
+    height: 800px;
+    width: 100%;
+    background-size: 100%, 100%;
+    padding-top: 20%;
+    text-align: center;
+    font-size: 50px;
+    color: white;
+    font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
```

Comment:
Since you're using this line of code multiple times, you can just set global fonts for the whole page using `*` global selector:
```
* {
 font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', 
   Arial, sans-serif;
}
```